tests/redirects/base.py

```diff
@@ -101,8 +107,10 @@ def assert_valid_url(url, location=None, status_code=requests.codes.moved_perman
     :param resp_headers: Dict of headers expected in the response.
     :param base_url: Base URL for the site to test.
     :param query: Dict of expected query params in `location` URL.
+    :param allow_redirects: Boolean indicating whether redirects should be followed.
+    :param final_status_code: Expected status code after following any redirects.
     """
-    kwargs = {'allow_redirects': False}
+    kwargs = {'allow_redirects': allow_redirects}
```

Comment:
Ah. `allow_redirects` is from the `requests` lib. Still, in our test suite context I like `follow_redirects` better. Not a hard requirement though.